txscript/script.go

```diff
@@ -322,6 +328,8 @@ func countSigOpsV0(script []byte, precise bool) int {
 				numSigOps += MaxPubKeysPerMultiSig
 			}
 
+		// XXX add case for OP_TGEN here
+
```

Comment:
Surfacing this TODO.

Given how `countSigOpsV0` is implemented (looping through every op) , it should "just work" with OP_TGEN tagged outputs. Probably just adding a test case is sufficient.

Reply:
Fixed as well.